src/commonTest/kotlin/fr/acinq/eclair/channel/states/ClosingTestsCommon.kt

```diff
@@ -1724,7 +1762,13 @@ class ClosingTestsCommon : EclairTestSuite() {
                 val (alice1, actions1) = alice.process(ChannelEvent.ExecuteCommand(CMD_FORCECLOSE))
                 assertTrue(alice1 is Closing)
                 assertEquals(7, actions1.size)
-                assertTrue(actions1.contains(ChannelAction.Storage.StoreOutgoingAmount(alice1.commitments.localCommit.spec.toLocal)))
+
+                val channelBalance = alice1.commitments.localCommit.spec.toLocal
+                if (channelBalance > 0.msat) {
+                    val onChainPayment = actions1.filterIsInstance<ChannelAction.Storage.StoreChannelClosing>().firstOrNull()
+                    assertNotNull(onChainPayment)
+                    assertTrue(onChainPayment.amount == channelBalance)
```

Comment:
You should also validate the other fields of `StoreChannelClosing`.
There should be one test where we send to a non-default address as well.
And there should be a test validating the contents of `StoreChannelClosed` too.

Reply:
Done in 9353876